core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -744,9 +744,16 @@ public void sendTo(Address destination, ReplicableCommand rpcCommand, DeliverOrd
          }
          return;
       }
-      dispatcher.sendMessage(
+      CommandAwareRpcDispatcher rpcDispatcher = dispatcher;
+      if (rpcDispatcher == null) {
+         if (trace) {
+            log.trace("Don't send message. Transport is stopped.");
```

Comment:
you still need to duplicate the check since the method had to return. I don't think it would save anything...